packages/internal-postcss-fix-relative-paths/tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "../tsconfig.base.json",
+  "include": ["**/*.ts"],
```

Comment:
This allows for typechecking only the files inside this project.

Without it, it tries to type check the `**/*` relative to the base tsconfig.

This change should probably be made to all the tsconfig files 😅